### DISCUSSION TITLE
Hey feel free to discuss the topics mentioned in the community call here

### DISCUSSION THREAD

**Discussion by ikrishg** [EddieHubCommunity/support#8634] [Community Call]
- Introductions
- Topic Suggestions
- Comments on our topic
- Shout outs
- Todo lists from this topic

**Comment by ikrishg** [comment]
Feel free to drop yourself a introduction here (we love voice introductions even more though) 👇🏻

**Reply by thesumitkrsingh** [reply]
hey folks my name is  sumit currently student pursuing information technology and the passion and curiousity of tech pull me here 
i am very excited to know about these community  and collabrate with likeminded folks

**Comment by ikrishg** [comment]
Please drop your twitters here 👇🏻

**Reply by ikrishg** [reply]
@xkrishguptaa

**Reply by mocdaniel** [reply]
@d_bodky

**Reply by tamalCodes** [reply]
@tamalCodes

**Reply by hastyquickbeam** [reply]
@erikkrianderson
on Twitter

**Reply by Nitin6404** [reply]
@developer_nitin

**Reply by thesumitkrsingh** [reply]
@Thesumitkrsingh

**Comment by ikrishg** [comment]
Agendas:
- [x] Public Speaking
- [x] How to fend off questions that try to bring you down?
- [x] What is public speaking?
- [x] What does it really mean to be building in public?

**Comment by eddiejaoude** [comment]
Thanks Krish for organising and hosting!! Here is the tweet https://twitter.com/eddiejaoude/status/1751745713019465767
